cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -605,6 +614,10 @@ public TwillController start(long timeout, TimeUnit timeoutUnit) {
         }
       }
 
+      if (cdapRuntimeNamespace != null && !isSystemNamespace(cdapRuntimeNamespace)) {
```

Comment:
when is cdapRuntimeNamespace null and why do we need to do this check? Would be good to add a comment

Reply:
Checked with Dennis - null check is actually not needed, it was added just to be safe. I have removed it as it's unnecessary